generators/generator-base.js

```diff
@@ -1201,6 +1201,9 @@ module.exports = class extends PrivateBase {
     composeExternalModule(npmPackageName, subGen, options) {
         const generatorName = jhipsterUtils.packageNameToNamespace(npmPackageName);
         const generatorCallback = `${generatorName}:${subGen}`;
+        if (!this.env.get(generatorCallback)) {
+            throw new Error(`Generator ${generatorCallback} isn't registered.`);
+        }
```

Comment:
Is it blueprint-specific? I don't get why it's added

Reply:
composeWith is supposed to throw if the generator is not found.
But our blueprint implementation it to try to load every sub generator `blueprint:*`.
So this test verify if the sub generator exists before trying to load.